Samples/blazor-add-in/outlook-blazor-add-in/README.md

```diff
@@ -0,0 +1,131 @@
+---
+page_type: sample
+urlFragment: outlook-blazor-add-in
+products:
+  - office-add-ins
+  - office
+languages:
+  - javascript
+  - C#
+extensions:
+  contentType: samples
+  technologies: Add-ins
+  createdDate: '08/25/2022 11:13:00 PM'
+description: 'Create a Blazor Webassembly Outlook add-in showcasing some samples.'
+---
+
+# Create a Blazor Webassembly Outlook add-in
+
+This sample shows how to build an Outlook add-in using .NET Blazor technologies. Blazor Webassembly allows you to build Office Add-ins using .NET, C#, and JavaScript to interact with the Office JavaScript API. The add-in uses JavaScript to work with the document and Office JS APIs, but you build the user interface and all other non-Office interactions in C# and .NET Core Blazor technologies.
+
+Working with Blazor Webassembly in the framework of Office, you can:
+- Build cross-platform Office Add-ins using Blazor, C#, and JavaScript Interop
+- Initialize the Office JavaScript API library in Blazor context
+- Interact with Outlook to manipulate paragraphs and content controls
+- Interact with document content through Office JavaScript APIs
+
+## Applies to
+
+- Outlook on the web, Windows, and Mac.
+
+## Prerequisites
+
+- Microsoft 365 - Get a [free developer sandbox](https://developer.microsoft.com/microsoft-365/dev-program#Subscription) that provides a renewable 90-day Microsoft 365 E5 developer subscription.
+
+## Run the sample
+
+1. Download or clone the [Office Add-ins samples repository](https://github.com/OfficeDev/Office-Add-in-samples).
+1. Open Visual Studio 2022 and open the **Office-Add-in-samples\Samples\blazor-add-in\outlook-blazor-add-in\outlook-blazor-add-in.sln** solution.
+1. To run the demo, select Project **outlook-blazor-sideloader** and select **Office Desktop Client**.
+1. The first time you run the demo, enter the email address of the account you want to access in the **Email Address** property of the **outlook-blazor-sideloader** project. This sets the **Use multi-factor auth** to **true**.   
+1. To start the solution, choose **Debug** > **Start Debugging** or press **F5**.
+1. When Outlook opens, choose **Home** > **Show Taskpane**.
+
+Next, try out the controls.
+
+## Understand an Office Add-in in Blazor Context
+
+An Office Add-in is a web application that extends Office with additional functionality for the user. For example, an add-in can add ribbon buttons, a task pane, or a content pane with the functionality you want. Because an Office Add-in is a web application, you must provide a web server to host the files.
```

Comment:
```suggestion
An Office Add-in is a web application that extends Office with additional functionality for the user. For example, Outlook add-ins can add ribbon buttons, and provide a task pane during compose or read modes. Because an Office Add-in is a web application, you must provide a web server to host the files.
```